lib/g-uni-v1-core/contracts/abstract/GUniFactoryStorage.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.19;
 import {OwnableUninitialized} from "./OwnableUninitialized.sol";
 import {
     Initializable
-} from "@openzeppelin/contracts-upgradeable/utils/Initializable.sol";
+} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
```

Comment:
There's a weird remapping with this file. When you compile it at a top-level, you have to remove the `proxy` portion, but it's required within the library to use the deployment via hardhat. Kind of annoying, but may be worth trying to get them to match up.

Reply:
> I think the build is failing because there are different versions of OZ being used and the file has moved. Will need to figure out how to fix this.

Yep. Will fix.

Reply:
Fixed.